internal/bake/hcl/parser/schema.go

```diff
@@ -120,150 +167,240 @@ var BakeSchema = &schema.BodySchema{
+							Kind:  lang.MarkdownKind,
+						},
 					},
 					"shm-size": {
 						IsOptional: true,
 						Constraint: schema.AnyExpression{OfType: cty.String},
+						Description: lang.MarkupContent{
+							Value: "Size of `/dev/shm` for RUN instructions. This has the same effect as passing [`--shm-size`](https://docs.docker.com/reference/cli/docker/buildx/build/#shm-size) flag to the build command.",
+							Kind:  lang.MarkdownKind,
+						},
 					},
 					"ssh": {
 						IsOptional: true,
 						Constraint: schema.List{Elem: schema.AnyExpression{OfType: cty.String}},
+						Description: lang.MarkupContent{
+							Value: "SSH agent sockets or keys to expose to the build. This has the same effect as passing [`--ssh`](https://docs.docker.com/reference/cli/docker/buildx/build/#ssh) flags to the build command.",
+							Kind:  lang.MarkdownKind,
+						},
 					},
 					"tags": {
 						IsOptional: true,
 						Constraint: schema.List{Elem: schema.AnyExpression{OfType: cty.String}},
+						Description: lang.MarkupContent{
+							Value: "Image names and tags for the built image. This has the same effect as passing [`--tag`](https://docs.docker.com/reference/cli/docker/buildx/build/#tag) flags to the build command.",
+							Kind:  lang.MarkdownKind,
+						},
 					},
 					"target": {
 						IsOptional: true,
 						Constraint: schema.AnyExpression{OfType: cty.String},
 						Description: lang.MarkupContent{
-							Value: "Set the target build stage to build. This is the same as the [`--target flag`](https://docs.docker.com/reference/cli/docker/image/build/#target).",
+							Value: "Set the target build stage to build. This has the same effect as passing [`--target`](https://docs.docker.com/reference/cli/docker/buildx/build/#target) flag to the build command.",
 							Kind:  lang.MarkdownKind,
 						},
 					},
 					"ulimits": {
 						IsOptional: true,
 						Constraint: schema.List{Elem: schema.AnyExpression{OfType: cty.String}},
+						Description: lang.MarkupContent{
+							Value: "Ulimit options for the build container. This has the same effect as passing [`--ulimit`](https://docs.docker.com/reference/cli/docker/buildx/build/#ulimit) flags to the build command.",
+							Kind:  lang.MarkdownKind,
+						},
 					},
 				},
 			},
 		},
 	},
-}
-
-type PathReaderImpl struct {
-	File     *hcl.File
-	Filename string
-}
-
-func (r *PathReaderImpl) Paths(ctx context.Context) []lang.Path {
-	return nil
-}
-
-func (r *PathReaderImpl) PathContext(path lang.Path) (*decoder.PathContext, error) {
-	return &decoder.PathContext{
-		Files:  map[string]*hcl.File{r.Filename: r.File},
-		Schema: BakeSchema,
-	}, nil
-}
-
-func ConvertToHCLPosition(content string, line, column int) hcl.Pos {
-	lines := strings.Split(content, "\n")
-	offset := 0
-	for i := range line {
-		offset += len(lines[i]) + 1
-	}
-	offset += column
-	return hcl.Pos{Line: line + 1, Column: column + 1, Byte: offset}
-}
```

Comment:
@Pletea-Marinescu-Valentin Did you intend to delete this code? The project no longer compiles without this and the tests are failing.